test/xbgpu/test_engine.py

```diff
@@ -729,6 +754,26 @@ def _make_feng(
         )
         return spead2.send.asyncio.InprocStream(spead2.ThreadPool(), queues, feng_stream_config)
 
+    @staticmethod
+    async def get_data_heap(stream: spead2.recv.asyncio.Stream, ig_recv: spead2.ItemGroup) -> set[str]:
+        r"""Receive heaps from a stream until a non-descriptor heap arrives.
+
+        This modifies `ig_recv` in place in order to obtain an updated set of
+        :class:`spead2.Item`\ s.
+
+        Parameters
+        ----------
+        stream
+            Streams that received XBEngine
```

Comment:
The parameter list is incomplete. The return value is also not documented (and probably needs it more than the `stream` parameter).